samples/algorithms/Grover.qs

```diff
@@ -22,15 +22,16 @@ operation Main() : Result[] {
     // Grover's algorithm relies on performing a "Grover iteration" an
     // optimal number of times to maximize the probability of finding the
     // value we are searching for.
-    // You can set the number iterations to a value lower than optimal to
-    // intentionally reduce precision.
-    let iterations = CalculateOptimalIterations(nQubits);
-    Message($"Number of iterations: {iterations}");
+    // You can intentionally set the number iterations to a value lower
```

Comment:
Do we want to say "lower or higher" here, since _any_ divergence from this number will reduce the probability of finding the right answer?

Reply:
There's not much value in making this number higher - you'll use more computation for nothing - so I think it's not a good idea to suggest this. Besides, some higher numbers may bring you to the right solutions after going around the circle.
If you are curious, there's another comment on the `IterationsToMarked` function, which tells you what happens if the number of iterations is different. So I think I should leave this comment as is.